client/human/HumanClientApp.cpp

```diff
@@ -687,7 +687,8 @@ void HumanClientApp::MultiPlayerGame() {
             if (!GetOptionsDB().OptionExists(cookie_option + ".cookie"))
                 GetOptionsDB().Add<std::string>(cookie_option + ".cookie", "OPTIONS_DB_SERVER_COOKIE", "");
             if (!GetOptionsDB().OptionExists(cookie_option + ".address"))
-                GetOptionsDB().Add<std::string>(cookie_option + ".address", "OPTIONS_DB_SERVER_COOKIE", server_dest);
+                GetOptionsDB().Add<std::string>(cookie_option + ".address", "OPTIONS_DB_SERVER_COOKIE", "");
```

Comment:
why just `""` here but `boost::uuids::to_string(boost::uuids::nil_uuid())` on the client?

Reply:
`server_dest` isn't uuid value but server host address. I suppose empty for addresses and `nil_uuid` for uuids are reasonable default values.